content/documentation/glossary/concepts/index.adoc

```diff
@@ -0,0 +1,94 @@
+---
+title: "Concepts"
+date: 2018-06-20T19:04:38+02:00
+draft: false
+type: "documentation/glossary"
+---
+
+= Kiali Concepts Glossary
+:sectnums:
+:toc: left
+toc::[]
+:toc-title: Terminology Content
+:keywords: Kiali Documentation
+:icons: font
+:imagesdir: /images/documentation/terminology/
+
+== Application
+
+Is a logical grouping of <<Workload>>s defined by the application labels that users apply to an object. In Istio it is defined by the <<Label App>>. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements].
+
+=== Application Name
+
+It's the name of the <<Application>> deployed in your environment. This name is provided by the <<Label App>> on the <<Workload>>.
+
+
+== Deployment
+
+A deployment is a replication controller based on a user defined template called a <<Deployment>> configuration. Deployments are created manually or in response to triggered events.
+
+=== Istio object/configuration Type
+
+This is the type specified in the <<Istio Config>>. This could be any of the following types: <<Gateway>>, <<VirtualService>>, <<DestinationRule>>, <<ServiceEntry>>, <<Rule>>, <<QuotaSpec>> or <<QuotaSpecBinding>>.
+
+== Istio Sidecar
+
+For more information see the <<Istio Sidecar>> definition in link:https://istio.io/docs/reference/commands/sidecar-injector/[Istio Sidecar Documentation].
+
+== Quota
+
+A limited or fixed number or amount of resources.
+
+== QuotaSpecBinding
+
+Quota spec binding is used to bind link:https://istio.io/help/faq/mixer/[mixer] quota resource name to a proxy.
+
+== Label
+
+It's a user-created tag to identify a set of objects.
+
+An empty link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (that is, one with zero requirements) selects every object in the collection.
+
+A null link:https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/[label selector] (which is only possible for optional selector fields) selects no objects.
+
+For example, Istio uses the <<Label App>> & <<Label Version>> on a <<Workload>> to specify the version and the application.
+
+=== Label App
+
+This is the 'app' label on an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements].
+
+=== Label Version
+
+This is the 'version' label on an object. link:https://istio.io/docs/setup/kubernetes/spec-requirements/[Istio Label Requirements].
+
+
+== Namespace
+
+Namespaces are intended for use in environments with many users spread across multiple teams, or projects.
+
+Namespaces are a way to divide cluster resources between multiple users.
+
+== ReplicaSet
+
+Ensures that a specified number of pod replicas are running at any one time.
+
+== Service
+
+Pods are dynamically created, they are ephemeral. A <<Service>> is an abstraction which defines a logical set of Pods and a policy by which to access them.
```

Comment:
I'd remove the first sentence, it doesn't explain much and seems out of place (they'd be great on a `Pod` definition, but not here).